administrator/components/com_admin/sql/updates/mysql/4.1.0-2021-07-16.sql

```diff
@@ -41,17 +41,19 @@ CREATE TABLE IF NOT EXISTS `#__guidedtour_steps` (
   `published` tinyint NOT NULL DEFAULT 0,
   `description` text NOT NULL,
   `ordering` int NOT NULL DEFAULT 0,
-  `created` datetime NOT NULL,
+  `step-no` int NOT NULL DEFAULT 0,
+  `position` varchar(255) NOT NULL,
+  `target` varchar(255) NOT NULL,
+  `offset` varchar(255) NOT NULL,
+  `url` varchar(255) NOT NULL,
+  `overlay` tinyint NOT NULL  DEFAULT 0,
+  `created` datetime NOT NULL DEFAULT 0,
   `created_by` int unsigned NOT NULL DEFAULT 0,
-  `modified` datetime NOT NULL,
+  `modified` datetime NOT NULL DEFAULT 0,
```

Comment:
A default value of zero for a datetime column does not work, at least not if the server is running in strict mode. Please look up in other create table statements of Joomla CMS how the `created`and `modified` columns are defined there.

Reply:
The same applies to all other columns of type datetime, also in the SQl script for new installations.